src/pattern-library/components/Library.tsx

```diff
@@ -64,7 +60,7 @@ function Heading({
   if (level <= 2) {
     return (
       <h2
-        className="text-3xl text-slate-600 font-bold mb-4"
+        className="text-2xl text-slate-600 font-medium mb-4"
```

Comment:
Demoted the size of headings one font-size tick to play more nicely with the updated page heading.